deployments/helm/dra-example-driver/templates/kubeletplugin.yaml

```
@@ -53,6 +53,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: NUM_DEVICES
```

```suggestion
        # Simulated number of devices the example driver will pretend to have.
        - name: NUM_DEVICES
```
Without context this might be misunderstood, how about a comment ?